src/TSTransformer/util/cleanModuleName.ts

```diff
@@ -1,3 +1,3 @@
 export function cleanModuleName(name: string) {
-	return name.replace(/[\~\-\.]/g, "_");
+	return name.replace(/[\~\-\.@]/g, "_");
```

Comment:
As mentioned in https://github.com/roblox-ts/roblox-ts/issues/1469, maybe this should be 

```ts
name.replace(/\W/g, "_")
```
Would ensure there isn't any other bugs with any other special characters. This is basically "anything not alpha-numeric.". Underscores should be fine still.